components/channel_notifications_modal/channel_notifications_modal.jsx

```diff
@@ -283,8 +306,9 @@ export default class ChannelNotificationsModal extends React.PureComponent {
                                         expand={activeSection === NotificationSections.DESKTOP}
                                         memberNotificationLevel={desktopNotifyLevel}
                                         globalNotificationLevel={currentUser.notify_props ? currentUser.notify_props.desktop : NotificationLevels.ALL}
+                                        desktopNotification={desktopNotification}
```

Comment:
Since `DesktopNotificationSection` works so differently from `NotificationSection`, maybe it would be a better idea to use `DesktopNotificationSection` here directly instead of having it be a child of `NotificationSection` so that we're not adding an extra case to `NotificationSection` that behaves entirely differently and requires passing around extra props that are only used in that one case